app/controllers/converts_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ConvertsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    params.require(:file)
+    params.require(:format)
+    run_file = RunFile.for_convert(params[:file])
+    @run = Run.new(run_file: run_file, user: nil)
+    @run.parse(fast: ["on", "1"].include?(params[:historic]) ? false : true, convert: true)
+    unless @run.splits
+      redirect_to cant_parse and return
```

Comment:
I roughly follow [bbatsov/ruby-style-guide](https://github.com/bbatsov/ruby-style-guide#no-and-or-or) which says to never use `and`. Just use two lines or `&&`.